### PR TITLE
Milestone 5: Event handlers

### PR DESCRIPTION
**Pull request by nicolasmarino99**
### In this Milestone we did:

- A click handler and state to App
- A click handler for each button
- Change button to class structure for receiving props
- Add logic to each calculate button
- Fix linter errors

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
-  name, wide = false, color = 'orange',
-}) => {
-  let sizeClass;
-  // eslint-disable-next-line no-unused-expressions
-  wide ? sizeClass = 'big-btn' : sizeClass = 'small-btn';
-  const colorClass = `${color}`;
-  // eslint-disable-next-line react/button-has-type
-  return <button className={`Button ${sizeClass} ${colorClass}`}>{name}</button>;
-};
+class Button extends Component {
```
Comment (Oghenebrume50) [inline src/components/Button.js]: Components like this that do not have state should not be a class component, make them class components

File: src/logic/operate.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-      return num1.times(num2);
+      return num1.times(num2).toString();
     case '÷':
-      if (num2 === 0) { return '∞'; }
-      return num1.div(num2);
+      if (num2.eq(0)) { return '∞'; }
+      return num1.div(num2).toString();
     case '%':
-      return num1.mod(num2);
+      return num1.mod(num2).toString();
```
Comment (Oghenebrume50) [inline src/logic/operate.js]: this shoud be the percentage of the num1 only, 
see this 
so basically num1 / 100 should work perfectly

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @nicolasmarino99 awesome work done so far :+1: I have commented on some things that can be worked on below, but just to emphasize

- Use functional component if a component has no state. 
- Work on handling the percentage operator, it currently returns an error. 

- To deal with some of the linter errors, use de-structuring where possible

please make changes and request for another review :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask: stay_safe

**Review by jofer86 — APPROVED** [review]
## :wave: :wave: Hello!, I'm Jorge, and  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- All of the required changes were met!!
## `const STATUS = 'Approved'` 🚀 🚀 🚀
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

**Pull request merged**